### DISCUSSION TITLE
I run build and start, now what?

### DISCUSSION THREAD

**Discussion by rvyh** [vercel/next.js#20943] [Help] [answered]
I cloned my project from GitHub to my VPS running Node and nginx. I run build and start, after which it says in the console:

```
npm run start

> myproject@0.1.0 start /root/myproject
> next start

ready - started server on http://localhost:3000
```

I managed to configure nginx as reverse proxy, but now **how do I keep my app online even after server restart?** Is it possible to use pm2 for this? If yes, how?

**Comment by jamesmosier** [comment, marked as answer]
Yes, pm2 would be a great option. See these Discussions for some options on how to do it: https://github.com/vercel/next.js/discussions/19728 and https://github.com/vercel/next.js/discussions/10675

```bash
pm2 start npm --name "myApplicationName" -- run start
```

**Reply by rvyh** [reply]
Thanks a lot! it worked!

**Comment by qburst-sachin** [comment]
> I cloned my project from GitHub to my VPS running Node and nginx. I run build and start, after which it says in the console:
> 
> ```
> npm run start
> 
> > myproject@0.1.0 start /root/myproject
> > next start
> 
> ready - started server on http://localhost:3000
> ```
> 
> I managed to configure nginx as reverse proxy, but now **how do I keep my app online even after server restart?** Is it possible to use pm2 for this? If yes, how?

H @jamesmosier ,@rvyh , 
Can you kindly share how did you serve the content with nginx. 

Currently what I am doing is that I ran npm run build and then copied the /out folder to nginx root folder and served it as static content.

Is this the right approach or should I do anything else. ?
